airflow/providers/google/__init__.py

```diff
@@ -14,3 +14,19 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+import logging
+
+from airflow.configuration import conf
+
+PROVIDERS_GOOGLE_VERBOSE_LOGGING: bool = conf.getboolean('providers_google',
+                                                         'VERBOSE_LOGGING', fallback=False)
+if PROVIDERS_GOOGLE_VERBOSE_LOGGING:
+    for a in ["google_auth_httplib2", "httplib2", "googleapiclient"]:
+        logger = logging.getLogger(a)
+        logger.handlers = [handler for handler in
```

Comment:
Should we add instead of replace handlers?

Reply:
What would be the consequence of that? Do you think it's better to add?

Reply:
I configure loggers in my configuration file. Now, when I use this option, my configuration will be overwritten. I won't even know why my configuration has not been used because there will be no message.This is a very small problem.

Reply:
Now I understand why. Updated.